.codex/skills/zammad-mcp-pr-digest/scripts/collect_pr_digest.py

```
@@ -0,0 +1,186 @@
+            "--state",
+            "all",
+            "--limit",
+            str(limit),
+            "--search",
+            f"updated:>={since_date}",
+            "--json",
+            PR_LIST_FIELDS,
+        ]
+    )
+    if not isinstance(payload, list):
+        raise GhCommandError("Unexpected PR list payload")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)